### PR TITLE
 ENT-8980: The prompt for adding all modules now has yes/y/no/n options, like the other prompts

### PR DESCRIPTION
**Pull request by danielskinstad**
Changed hardcoded print of [y/N] to [yes/y/NO/n] using helper function to be consistent

Ticket: ENT-8980

### CODE REVIEW

File: cfbs/prompts.py
Comment (larsewi) [inline cfbs/prompts.py:37]: This should be on a separate commit, please see https://github.com/cfengine/core/blob/master/CONTRIBUTING.md#commits

**Review by larsewi — CHANGES_REQUESTED** [review]
Thank you for your contribution 🚀 

Before merging please:
 - [ ] put different unrelated changes in their own commits
 - [ ] format code with the [black](https://pypi.org/project/black/) formatter for python
 - [ ] sign your commits using `git commit -s` (see https://github.com/cfengine/core/blob/master/CONTRIBUTING.md#commit-message-example)
 - [ ] add the ticket number to the PR Title (see https://github.com/cfengine/core/blob/master/CONTRIBUTING.md#pull-request-title-and-description)

Also I would recommend not committing directly to you master branch, as it can make some things a little bit more complicated for you down the line. Instead create a new branch to work on.

To do all the fixes above I would recommend getting to know how to do an interactive rebase (see https://github.com/cfengine/core/blob/master/CONTRIBUTING.md#interactive-rebase)

Lastly, don't hesitate to bombard me with questions!

**Review by olehermanse — CHANGES_REQUESTED** [review]
The change itself looks good, the commits we can improve. In addition to the things mentioned by @larsewi, it also looks like you have committed as "cfbs" - please put your name and email as author / committer when you make commits.

Furthermore, it would be better if the PR title / commit title focused on what is the impact to a user. Here is my suggestion:

```
The prompt for adding all modules now has yes/y/no/n options, like the other prompts
```

Feel free to ask me for help if you are unsure about how to edit commits :)

**Review by olehermanse — APPROVED** [review]
Looks great now, thanks! 🚀

**Comment by larsewi** [comment]
Congrats 🚀

**Pull request merged**
